components/form/form-element-localize-helper.js

```diff
@@ -1,36 +1,12 @@
 import { formatNumber } from '@brightspace-ui/intl/lib/number.js';
```

Comment:
The diff for this is horrible, but all the helpers were moved above the exported const that uses them